src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCenterAddForm/MedCenterAddForm.tsx

```diff
@@ -0,0 +1,21 @@
+import Input from '@components/common/Input/Input';
+import { FormName } from '@components/Form/SignInUp/FormStyle';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { MedCenterFormWrapper } from './MedCenterAddFormStyle';
+
+const MedCenterForm = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+  return (
+    <MedCenterFormWrapper>
+      <FormName>Edit or create new medical center</FormName>
+      <Input primary label="Email" name="email" register={register('email')} errors={errors} />
```

Comment:
is there gonna be only one field in the form for centers?

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/26/commits/6a7aa338bd2cf39a43f0bb25411635179e962756#diff-82c3df5933f2e8057602e5ffc8dca1781e551a9ea622c252f31d3ac117e2fe1f